src/amo/components/AddonReviewList.js

```diff
@@ -141,6 +142,18 @@ export class AddonReviewListBase extends React.Component {
     const {
       addon, errorHandler, location, params, i18n, reviewCount, reviews,
     } = this.props;
+
+    if (errorHandler.hasError()) {
+      log.warn('Captured API Error:', errorHandler.capturedError);
+      // A 401 is made to look like a 404 on purpose.
+      // See https://github.com/mozilla/addons-frontend/issues/3061
+      if (errorHandler.capturedError.responseStatusCode === 401 ||
```

Comment:
That's for an add-on, but will requesting reviews for an add-on that would 401 also send a 401 unauthorised? I guess it's likely, maybe @diox knows off the top of his head.

I guess we should also 404 here then.

Reply:
Even I had that confusion, that's why I put both check :)